requirements.txt

```diff
@@ -1,2 +1,3 @@
 numpy
 pandas
+tqdm
```

Comment:
I have deleted this file and use `pipenv` for managing dependencies. Could you rebase the master branch and add `tqdm` to the 'requirements' in `setup.py`?